pygin/pygin/src/helpers.hpp

```diff
@@ -45,4 +46,30 @@ namespace helpers
 			return Result;
 		}
 	}
+
+	class py_string_storage
+	{
+	public:
+		py_string_storage() {}
+
+		py_string_storage(py_string_storage const&) = delete;
```

Comment:
There's a [macro](https://github.com/trexinc/evil-programmers/blob/5a53e2c4ca9c7daacb443528227c8ae3e65b50a3/pygin/pygin/src/headers.hpp#L57-L59).

Reply:
Fixed in 5ef1f612ff8787268b8d48658e3d1e8cad9b7272.